internal/commands/app/create_test.go

```diff
@@ -230,222 +230,250 @@ func TestAppCreateHandler(t *testing.T) {
 		}, "\n"), out.String())
 	})
 
-	t.Run("should create minimal project with data source when data source is set", func(t *testing.T) {
-		profile, teardown := mock.NewProfileFromTmpDir(t, "app_create_test")
-		defer teardown()
-		profile.SetRealmBaseURL("http://localhost:8080")
+	for _, tc := range []struct {
+		description string
+		cluster     string
+		dataLake    string
+		atlasClient atlas.Client
+	}{
+		{
```

Comment:
[question] do we (care to) test the case where neither `cluster` nor `dataLake` are provided here?

Reply:
Those cases are above and different enough not to be in the table